.travis.yml

```diff
@@ -10,7 +10,7 @@ notifications:
 #   - travis-build@platform-notifications.robomaker.aws.a2z.com  #Uncomment once the build passes
 env:
   matrix:
```

Comment:
Can you please also add a deployment stage? make sure you structure it like in hello-world where we add a separate deploy stage rather than doing it as part of each workspace's build: https://github.com/aws-robotics/aws-robomaker-sample-application-helloworld/blob/ros2/.travis.yml#L34

Resulting build example: https://travis-ci.org/aws-robotics/aws-robomaker-sample-application-helloworld/builds/589233448

Reply:
Done.